client/my-sites/importer/site-importer/site-importer-input-pane.jsx

```diff
@@ -228,10 +230,8 @@ class SiteImporterInputPane extends React.Component {
 				);
 			case 'godaddy-gocentral':
 				return (
-					<div>
-						<p>
-							{ this.props.translate( 'Please use one of following formats for the site URL:' ) }
-						</p>
+					<div className="site-importer__site-importer-url-format">
+						<p>{ this.props.translate( 'Please use the following format for the site URL:' ) }</p>
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.